src/toggle-switch/toggle-switch.scss

```diff
@@ -2,194 +2,205 @@
-        border: 1px solid rgba(t(iui-color-foreground-body-rgb), t(iui-opacity-4));
-      }
+  // Disabled
+  &.iui-disabled {
+    cursor: not-allowed;
+    @include themed {
+      color: t(iui-text-color-muted);
+    }
+  }
 
-      > .iui-icon {
-        opacity: 0;
-        width: $iui-sm;
-        height: $iui-sm;
-        display: inline-block;
-        position: relative;
-        left: $iui-xs;
-        @media (prefers-reduced-motion: no-preference) {
-          transition: opacity $iui-speed-fast ease;
-        }
-        @include themed {
-          fill: t(iui-color-foreground-accessory);
-        }
+  // For labels on the left
+  &.iui-label-on-left {
+    grid-template-areas: 'label toggle';
+
+    .iui-label {
+      margin-left: $iui-s;
+      margin-right: 0;
+      @supports (gap: $iui-s) {
+        margin-left: 0;
       }
+    }
+  }
 
-      > .iui-handle {
-        position: absolute;
-        height: $iui-m;
-        width: $iui-m;
-        top: 50%;
-        transform: translateY(-50%);
-        right: $iui-m + $iui-xs;
-        border-radius: 50%;
-        @media (prefers-reduced-motion: no-preference) {
-          transition: right $iui-speed-fast ease, background-color $iui-speed-fast ease, opacity $iui-speed-fast ease;
-        }
-        @include themed {
-          background-color: t(iui-color-foreground-body);
-          opacity: t(iui-opacity-2);
-        }
+  // For labels on the right
+  &.iui-label-on-right {
+    grid-template-areas: 'toggle label';
+
+    .iui-label {
+      grid-area: label;
+      margin-right: $iui-s;
+      @supports (gap: $iui-s) {
+        margin-right: 0;
       }
     }
+  }
+}
 
-    &:hover:not(:disabled),
-    &:focus {
-      ~ .iui-toggle {
-        @include themed {
-          border: 1px solid rgba(t(iui-color-foreground-body-rgb), t(iui-opacity-2));
-        }
+@mixin iui-toggle-switch {
+  $iui-toggle-switch-handle-size: $iui-icons-default;
+  $iui-toggle-switch-handle-offset: $iui-component-offset;
+  $iui-toggle-switch-border-thickness: 1px;
 
-        > .iui-handle {
-          @include themed {
-            opacity: t(iui-opacity-1);
-          }
-        }
-      }
+  @include iui-reset;
+  @include iui-focus($thickness: $iui-toggle-switch-border-thickness, $offset: $iui-toggle-switch-border-thickness);
+  grid-area: toggle;
+  display: flex;
+  position: relative;
+  cursor: pointer;
+  -webkit-tap-highlight-color: transparent;
+
+  // #region IE solution
+  width: $iui-toggle-switch-handle-size;
+  height: $iui-toggle-switch-handle-size;
+  @supports (appearance: none) {
+    appearance: none;
+    width: ($iui-toggle-switch-handle-size + ($iui-toggle-switch-handle-offset * 2)) * 2;
+    height: auto;
+    border-radius: $iui-border-radius-round;
+    box-sizing: border-box;
+    transition: background-color $iui-speed-fast ease-out, border-color $iui-speed-fast ease-out;
+    @include themed {
+      background-color: t(iui-color-background-1);
```

Comment:
Because there is no IE support, I think pretty much everything in this component could reference the CSS `var`s directly without the `themed` mixin or `t`.

Reply:
Removed `@include themed {` from all places _except_ `color` applied to `<label>` because that part does still work in IE.